### DISCUSSION TITLE
Attempt to read property "layout" on array

### DISCUSSION THREAD

**Discussion by kenmdavids** [Power-Components/livewire-powergrid#1426] [General]
After installing PowerGrid, and I include the component using the tag: <livewire:activities-ta
ble/>
It throws an error
`<div class="flex flex-col">

    <div class="-my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">

        <div class="py-2 align-middle inline-block min-w-full w-full sm:px-6 lg:px-8">

            @include($theme->layout->header, [  //this here

                'enabledFilters' => $enabledFilters

            ])`
on line
`[s\vendor\livewire-powergrid\components\frameworks\tailwind\table-base.blade](line=5)`

**Comment by lsmedia** [comment]
Have you published the views of Powergrid? If so, delete them or update them to the new version.

Check `/resources/views/vendor`
